openpathsampling/high_level/move_scheme.py

```diff
@@ -721,6 +722,10 @@ def move_summary(self, steps, movers=None, output=sys.stdout, depth=0):
         depth : integer or None
             depth of submovers to show: if integer, shows that many
             submovers for each move; if None, shows all submovers
+        scheme_copies : lits of :class:`MoveScheme`
```

Comment:
Mainly, my thoughts go like this: A lot of the high-level objects were originally designed to group data for analysis. While writing the analysis scripts, I realized that these would also make it really easy to set up a simulation. So analysis got combined with setup.

That's what should change. The setup side of all of these should stay the same, but there should be another object that takes a `MoveScheme` or a `TransitionNetwork` and actually provides the analysis. Then the `MoveScheme` does the setup, and the `MoveSchemeAnalyzer` does the analysis. The difficulties you're having would then be trivially solved with `MoveSchemeAnalyzer(schemes=[scheme1, scheme2])`.

I think I'll keep the current behavior available as a convenience (and for backward compatibility). For example, is it really nice to write `network.rate_matrix(steps)`, but it really should be `TISNetworkAnalyzer(network).rate_matrix(steps)`. (By adding an `analyzer` object to the network, it should be pretty easy to maintain the conveniences.) This would also make it easier to have multiple methods to calculate the rate (not just questions of WHAM/MBAR, but also path-type analysis, which is what we should be using.) This could be `1.1` or `1.0`; depends on how quickly we write the paper ;-)